classes/wrapper.oeclass

```diff
@@ -0,0 +1,55 @@
+## Class for packages having wrapper files.
+##
+## Rewrites the paths in the wrapper files so that it contains valid dirs.
+
+require conf/meta.conf
+
+WRAPPER_FILES ?= "${bindir}/*"
+WRAPPER_GLOB ?= "${WRAPPER_FILES}"
+
+DO_SPLIT_WRAPPER_FIXUP = ""
+DO_SPLIT_WRAPPER_FIXUP:native = "do_split_wrapper_fixup"
+do_split[postfuncs] += "${DO_SPLIT_WRAPPER_FIXUP}"
+def do_split_wrapper_fixup(d):
+    import oelite.path
+
+    pkgd = d.get("PKGD")
+    pkgmetadir = d.get("pkgmetadir").lstrip("/")
+    wrapper_globs = d.get("WRAPPER_GLOB").lstrip("/").split()
+    os.chdir(pkgd)
+    for pkg in os.listdir("."):
+        os.chdir(os.path.join(pkgd, pkg))
+        wrapper_files = []
+        for wrapper_glob in wrapper_globs:
+            for wrapper_file in glob.glob(wrapper_glob.lstrip("/")):
+                wrapper_dotpath = "%s/.%s"%(os.path.dirname(wrapper_file), os.path.basename(wrapper_file))
+   
+                localpath = oelite.path.which(d.get("FILESPATH_EXISTS"), wrapper_file)
```

Comment:
What is going on here?  Are you looking back at files in the manifest in a do_split postfunc?

In do_split you should only be concerned with the files in PKGD.